test/dune

```diff
@@ -3,5 +3,5 @@
     (public_name skia-test)
     (flags (:standard (-w -39)))
     (ocamlopt_flags -linkall -g)
-    (libraries rely.lib skia)
+    (libraries rely.lib rely.internal skia)
```

Comment:
This is a good example of the issue with module aliases. Without `rely.internal` we'll get the error:

```
File "test/TestFramework.re", line 3, characters 4-39:
Error: The module Rely.TestFrameworkConfig is an alias for module RelyInternal__RelyAPI.TestFrameworkConfig, which is missing
```
and this is despite the `TestFramework` interface being explicitly defined here:

https://github.com/facebookexperimental/reason-native/blob/e57f9054e3bba2e20bc920049797443667ad0cb7/src/rely/RelyAPI.rei#L213-L217

but it might be that the `include` of `RelyAPI` in `Rely` without an explicit interface causes problems:

https://github.com/facebookexperimental/reason-native/blob/e57f9054e3bba2e20bc920049797443667ad0cb7/src/rely/Rely.re#L1